### ISSUE TITLE
Crashes upon opening exe file regardless of where it is/ran as admin or not

### ISSUE THREAD

**Issue by Howard-F** [Epix37/Hearthstone-Deck-Tracker#781]
Title. Tried re installing as well, when opened it shows "The publisher could not be verified...." box which of course i hit run for, then windows pops open an error message "Hearthstone Deck Tracker has stopped working"

**Comment by Epix37** [comment]
Do you have .Net Framework 4.5 installed?

**Comment by Howard-F** [comment]
I have 4.0 installed i believe. Do you need 4.5?

**Comment by Epix37** [comment]
Yes

**Comment by Howard-F** [comment]
So I fixed this issue with this and felt pretty silly that it was something simple and within instructions, However i ran into a different issue now. I was able to open HDT and create a deck and save it, as well as have it show up when i open hearthstone. But, that is the extent of this, it does nothing else past this and when i play the same deck in game nothing is tracked. It is the exact same issue as #369. I tried installing it on my laptop as well but no avail and i run into the same problem. I looked for a config.xml file and i seem to be lacking one to make the change you suggested as a fix in the other thread as well.
